images/refinerycms-images.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.test_files        = `git ls-files -- spec/*`.split("\n")
 
   s.add_dependency 'refinerycms-dragonfly',   '~> 1.0'
-  s.add_dependency 'globalize',               ['>= 5.1.0.beta1', '< 5.2']
+  s.add_dependency 'mobility',               '~> 0.5'
```

Comment:
Isn't it a bit strange to have the mobility dependency in `refinerycms-images` ? I had trouble tracking down where it was. I would expect it to be in `refinerycms-i18n`.

Reply:
You're right, it's a bit strange, i think it's because `refinerycms-i18n` was not a dependency of `refinerycms-core` a long time ago.

We could add a dependency of `mobility` gem in `refinerycms-i18n`.

What do you think @parndt?

Reply:
Unless I'm mistaken it's because `refinerycms-images` doesn't depend on `refinerycms-i18n` so in order for images to be translated it has to depend on the translation library?

Reply:
> Unless I'm mistaken it's because refinerycms-images doesn't depend on refinerycms-i18n

You're right

> so in order for images to be translated it has to depend on the translation library?

You're right but `refinerycms-core` depends to `refinerycms-i18n`:
https://github.com/refinery/refinerycms/blob/90a44fc0ed08f541d285e0973670f4c21850c593/core/refinerycms-core.gemspec#L25

And `refinerycms-images` depends to `refinerycms-core`.

So what should we do @parndt?

Reply:
I'd go with @shioyama's suggestion

Reply:
> ah well in that case this library shouldn't duplicate dependencies

It does not duplicate dependencies because `refinerycms-i18n` has no dependencies to `mobility` nor `globalize`: https://github.com/refinery/refinerycms-i18n/blob/master/refinerycms-i18n.gemspec#L19-L20

 even if we call Mobility in this gem: https://github.com/refinery/refinerycms-i18n/pull/76/files

So i will create a dependency to `mobility` in `refinerycms-i18n` like it should be.